src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -2611,6 +2612,49 @@ public bool RelativeMotionUnGrab()
             return ret;
         }
 
+        /// <summary>
+        /// Sets the window blur using window blur information.
+        /// 
+        /// It is designed to apply a blur effect to a window based on specified parameters. 
+        /// This supports different types of blurring effects, including blurring the window's background only.
+        /// Or blurring the area surrounding the window while keeping the window itself clear.
+        /// The more information is written WindowBlurInfo struct.
+        /// </summary>
+        /// <param name="info"> Blur information for Window Blur.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetBlur(WindowBlurInfo info)
+        {
+            // Create Blur Info instance and binding.
+            IntPtr internalBlurInfo = Interop.WindowBlurInfo.New((int)info.BlurType, info.BlurRadius, info.BackgroundCornerRadius);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+            Interop.Window.SetBlur(SwigCPtr, internalBlurInfo);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+            Interop.WindowBlurInfo.DeleteWindowBlurInfo(internalBlurInfo);
+        }
+
+        /// <summary>
+        /// Gets the current window blur information.
+        /// </summary>
+        /// <returns>Returns current window blur information.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WindowBlurInfo GetBlur()
+        {
+            IntPtr internalBlurInfo = Interop.WindowBlurInfo.New();
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            
+            internalBlurInfo = Interop.Window.GetBlur(SwigCPtr);
```

Comment:
여기가 메모리 릭이네요..

`Interop.WindowBlurInfo.New();`한 값을 `internalBlurInfo`에 넣었는데, 
이를 사용하지 않고 (해제하지 않고)
GetBlur가 리턴하는 값을 internalBlurInfo에 덮어 썼기 때문에
`Interop.WindowBlurInfo.New();`를 통해 할당된 메모리 해제가 안되고 있습니다.

여기를 이렇게 바꾸면 될 것 같네요
```suggestion
            IntPtr internalBlurInfo = Interop.Window.GetBlur(SwigCPtr);
```

Reply:
네, 말씀해 주신 부분에 메모리 릭이 있었네요. Field 선언했던 부분은 삭제하고, Get/Set 로 변경했습니다.
좋은 리뷰 감사합니다. :)